### ISSUE TITLE
Check hook + genesis config block

### ISSUE THREAD

**Issue by daviddob** [genesis-community/prometheus-genesis-kit#17]
Check hook doesn't fail deployment with missing values - continues without failing

Also the new hook doesnt use the `genesis-config-block`

**Comment by dennisjbell** [comment]
Yes, the check script is broken.  After checking, it just sets check to okay, then if its okay, states the environment is ok.

https://github.com/genesis-community/prometheus-genesis-kit/blame/master/hooks/check#L11-L13
